src/components/Modals/AssetsFiltersModal/AssetsFiltersModal.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import { Rarity } from '@dcl/schemas'
+import { usePagination } from '../../../lib/pagination'
+import { NFTCategory, NFTOptions } from '../../../modules/nfts/types'
+import { MainCategory } from '../../../utils/categories'
+import { ITEMS_PER_PAGE } from '../../Assets/constants'
+import { NftFilters } from '../../NFTFilters'
+import FiltersModal from '../FiltersModal'
+import { Props } from './AssetsFiltersModal.types'
+import styles from './AssetsFilterModal.module.css'
+
+const AssetsFiltersModal = (props: Props) => {
```

Comment:
Would you mind adding a set of tests for this modal like the `Creations` one has?